.eslintrc

```diff
@@ -106,7 +106,8 @@
         "excludedForTesting.js"
       ],
       "rules": {
-        "oppia/e2e-action": "error"
+        "oppia/e2e-action": "error",
+        "oppia/nested-awaits": "error"
```

Comment:
Don't add this over here else this check won't run for all the above files, right?

Reply:
Yes, this is our normal practice for e2e lint checks. We didn't want new contributors to have to figure out how to make these changes throughout the e2e code since that can sometimes be tricky. Instead, we're applying the lint checks to a limited set of files for now, and then we'll let more experienced developers expand the covered files